Makefile

```diff
@@ -136,6 +136,17 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	$(CONTAINER_TOOL) push ${IMG}
 
+.PHONY: images-build
+images-build: test ## Build all container images
+	DPU_OPERATOR_IMAGE := $(shell yq -r .spec.template.spec.containers[0].image config/dev/local-images.yaml)
+	DPU_DAEMON_IMAGE := $(shell yq -r .spec.template.spec.containers[0].env[0].value config/dev/local-images.yaml)
+	$(CONTAINER_TOOL) build -v /tmp/go:/go:z -f Dockerfile.rhel -t $(DPU_OPERATOR_IMAGE)
+	$(CONTAINER_TOOL) build -v /tmp/go:/go:z -f Dockerfile.daemon.rhel $(DPU_DAEMON_IMAGE)
+
+.PHONY: images-push
+images-push: test ## Build all container images
+	$(CONTAINER_TOOL) push localhost:5000/dpu-operator
```

Comment:
the image names should come from the spec as well?

Reply:
done